Add Physical Measurements configs to aouRT underlay

dolbeew opened the pull request:
Add entities for blood pressure, bmi, heart rate, height, hip circumference, waist circumference and weight.

![Screenshot 2024-02-06 at 11 37 55 PM](https://github.com/DataBiosphere/tanagra/assets/40036095/55e93e30-41ea-4f19-ad2c-1a6a18fd41a5)

chenchals commented on underlay/src/main/resources/config/datamapping/aouRT/entity/bloodPressureOccurrence/all.sql:
Great!

chenchals reviewed (changes requested):
Need the following changes:
- I did not see indexed tables for `SC2023Q3R2` dataset for physical measurements 
  -  underlay/src/main/resources/config/underlay/aouSC2023Q3R2/ui.json
  - underlay/src/main/resources/config/underlay/aouSR2023Q3R2/underlay.json
- Run indexing build for Controlled tier also 
- I am guessing, you have deployed to local docker in AoU to check the UI

chenchals commented on underlay/src/main/resources/config/datamapping/aouRT/entity/bloodPressureOccurrence/entity.json:
```diff
@@ -0,0 +1,17 @@
+{
+  "name": "bloodPressure",
```
change to
"name": "bloodPressureOccurrence",

freemabd reviewed (approved):
Looks good. Just a few comments on folder names.

chenchals reviewed (changes requested):
Need to have changes to be added/modified for physical measurements
- underlay/aouSC2023Q3R2/ui.json
- underlay/aouSC2023Q3R2/underlay.json

dolbeew commented:
I renamed all entity folders to drop 'Occurrence.' Also added all entities and groups to the aouCT underlay and tested locally in the Workbench after running indexing.

chenchals reviewed (approved):
Thank you for the changes!
